.changeset/nice-snakes-camp.md

```diff
@@ -0,0 +1,90 @@
+---
+"@salt-ds/core": minor
+---
+
+**_Theming and CSS updates_** with visual changes to Salt components:
+
+1. Minor changes in `Button` component width. `CTA` and `Primary` button text font weight changed from bold to semibold. Expect components containing Salt `Button` to have similar change.
+
+```diff
+- --button-fontWeight: var(--salt-actionable-cta-fontWeight);
+- --button-fontWeight: var(--salt-actionable-primary-fontWeight);
+- --button-fontWeight: var(--salt-actionable-secondary-fontWeight);
++ --button-fontWeight: var(--salt-text-action-fontWeight);
+```
+
+<!-- INSERT IMAGE SHOWCASING DIFFERENCES HERE -->
+
+2. Disabled `InteractableCard` component has border color updated from blue to grey.
+
+```diff
+- border-color: var(--saltCard-borderColor-disabled, var(--salt-accent-borderColor-disabled));
++ border-color: var(--saltCard-borderColor-disabled, var(--salt-container-primary-borderColor-disabled));
+
+```
+
+<!-- INSERT IMAGE SHOWCASING DIFFERENCES HERE -->
+
+3. `Text` component has two new variants: `Action` and `Notation`. See [storybook](https://storybook.saltdesignsystem.com/?path=/story/core-text-text-qa--all-variants-grid) for more information.
+
+```tsx
+<Text styleAs="notation">Lorem ipsum</Text>
+```
+
+```tsx
+<Notation>Lorem ipsum</Notation>
+```
+
+**_Theming and CSS updates_** with no visual change to Salt components, useful for teams overriding theme locally:
+
+- `Avatar` component CSS token name updated
+
+```diff
+- --salt-icon-size-base
++ --salt-size-icon
+```
+
+- `Pill` component CSS token names updated
+
+In `Pill.css`, these tokens have been renamed:
```

Comment:
the above tokens imo don't need to be listed, especially since it's in labs. feels unreadable to me and a bit unnecessary. would just say pill styles have changed

Reply:
oops they're supposed to be in labs cs, movng them over